coreneuron/nrniv/cellorder1.cpp

```diff
@@ -530,9 +530,9 @@ static void admin1(int ncell,
     // cellsize is the number of nodes in the cell not counting root.
     // nstride is the maximum cell size (not counting root)
     // stride[i] is the number of cells with an ith node.
-    firstnode = (int*)ecalloc_align(ncell, sizeof(int));
-    lastnode = (int*)ecalloc_align(ncell, sizeof(int));
-    cellsize = (int*)ecalloc_align(ncell, sizeof(int));
+    firstnode = new int[ncell];
```

Comment:
From the function name, I assume `ecalloc_align` initializes the allocated memory to 0. So I would:

```c++
firstnode = new int[ncell]();
lastnode = new int[ncell]();
cellsize = new int[ncell]();
```